package.json

```diff
@@ -35,13 +35,15 @@
     "dayjs": "^1.10.5",
     "discord.js": "^12.5.3",
     "dotenv": "^10.0.0",
+    "file-system": "^1.2.2",
```

Comment:
Tu utilises ce module pour quoi ? Je ne le vois jamais importé.
Le module `fs` que tu importes dans l'autre fichier est un module built-in donc pas besoin de télécharger autre chose :)

Reply:
C'est une erreur , je pensais qu'il fallait installer ce module pour utiliser fs

Reply:
Pense à faire `npm uninstall file-system` et a commit :)

Reply:
Faudrait le supprimer avec `npm uninstall file-system`